webapp/components/loading-state.js

```diff
@@ -4,46 +4,74 @@
  * found in the LICENSE file.
  */
 
-/*
-LoadingState is a behaviour component for indicating when information is
-still being loaded (generally, fetched).
-*/
-
-
+/**
+ * LoadingState is a behaviour component for indicating when information is
+ * still being loaded (generally, fetched).
+ */
 const LoadingState = (superClass) => class extends superClass {
   static get properties() {
     return {
+      /**
+       * The number of active loading operations.
+       */
       loadingCount: {
         type: Number,
         value: 0,
-        observer: 'loadingCountChanged',
+        observer: '_loadingCountChanged',
       },
+      /**
+       * Whether the component is currently loading data.
+       * Computed based on `loadingCount`.
+       */
       isLoading: {
         type: Boolean,
-        computed: 'computeIsLoading(loadingCount)',
-        notify: true,
```

Comment:
Note: check if removing notify: true will break anything

Reply:
isLoading isn't used as two-way data bindings in the codebase anywhere[1]. It is safe

[1] https://github.com/search?q=repo%3Aweb-platform-tests%2Fwpt.fyi%20isLoading&type=code